systemlink/clients/core/_jupyter_http_configuration.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+
+"""Implementation of JupyterHttpConfiguration."""
+
+import os
+
+from systemlink.clients import core
+
+
+class JupyterHttpConfiguration(core.HttpConfiguration):
```

Comment:
Do we need to add this class to the core [__init__.py](https://github.com/ni/nisystemlink-clients-python/blob/master/systemlink/clients/core/__init__.py), or is that a follow-up task?

Also, do we have a task to make [HttpConfigurationManager](https://github.com/ni/nisystemlink-clients-python/blob/master/systemlink/clients/core/_http_configuration_manager.py) use this class?